docs/usage.md

```diff
@@ -107,6 +108,16 @@ There are three ways to generate models for a JSON Schema document.
 
 The library expects the `$schema` property for the document to be set in order to understand the input format. By default, if no other inputs are detected, it defaults to `JSON Schema draft 7`. The process of interpreting a JSON Schema to a model can be read [here](./inputs/JSON_Schema.md).
 
+## Generate models from Avro Schema documents
+
+See the below example to get started with Avro Schema for generating models.
+
+- [Generate from an Avro Schema JS Object](../examples/avro-schema-from-object)
+
+The Avro input processor expects the `type` property, as per [Avro Schema specs](https://avro.apache.org/docs/1.11.1/specification/#schema-declaration), in the input object in order to proceed successfully.
+
+> Note: Currently, we support `record` datatype for generating the `Object Model`.
```

Comment:
Incorrect, we support enum as well, is there any other types that you can define with Avro that we dont support after merging this?

Reply:
My question is, why are you making this `Note`, and which cases of Avro are we NOT supporting since you highlight `record` here? 🤨

Reply:
In Avro Schema, a JSON Object is defined using one of the two datatypes: [`map`](https://avro.apache.org/docs/1.11.1/specification/#maps) and [`record`](https://avro.apache.org/docs/1.11.1/specification/#schema-record)

We do not handle the `map` datatype as I am not able to think of any use case. What would we been seeking in input object with single datatype. This is the reason I mentioned this explicitly in note.
![image](https://github.com/asyncapi/modelina/assets/96991785/1acb2f3b-f9d8-4e14-a3a5-93050997e97e)

Reply:
Maps can easily be converted to our DictionaryMetaModel, with key being StringMetaModel and the value what ever is defined 🙂

Reply:
Didn't know this could be converted to `DictionaryMetaModel`.

```json
{
   "name": "additionalInfo",
   "type": "map",
   "value": "string"
}
```

Just wanna check if there is any use case of `fixed` and `byte` datatype defined in Avro Schema or not.

Reply:
Why is the Avro specification so bad at explaining what is possible... Can you have the `value` be an array or record? 🤨 Or is only [primitive types](https://avro.apache.org/docs/1.11.1/specification/#primitive-types) allowed? 

`fixed` and `byte` I would convert to string models. We don't have official byte representation in the core models.

Reply:
I need to check for this non primitive value type. However in my opinion I don't think non primitive is supposed to be there else it would have been explicitly mentioned there as done for the fields attribute in record type 

https://avro.apache.org/docs/1.11.1/specification/#schema-record